### ISSUE TITLE
Error :cannot open path of the current working directory: Permission denied on mounted directories

### ISSUE THREAD

**Issue by jantigenie** [yurijmikhalevich/rclip#155]
unexpected error when executing 'rclip dog' i, a sshfs ou smb mounted drive (/home/user/mountpoint) 
Debian 1/snapd/rclip 1.11.0

cannot open path of the current working directory: Permission denied on mounted directories

**Comment by yurijmikhalevich** [comment]
Hi @jantigenie! Because `snap` limits app access to certain folders, you have to run `sudo snap connect rclip:removable-media` to let it access mounted drives. Can you let me know if this fixed the issue for you?

**Comment by jantigenie** [comment]
Hello yurijmikhalevich,
Many thanks for your reply.
Unfortunately I still have an error message on sshfs and cifs mounted drives : 
'cannot open path of the current working directory: Permission denied'
The current user has rw rights on the current folder.
Thanks in advance for helping !

**Comment by yurijmikhalevich** [comment]
@jantigenie, interesting. Let me try to reproduce it.

For now, as a workaround, can you see if `AppImage` version of rclip will work for you? It isn't being sandboxed by snap and should have access to all directories the Linux user has access to.

**Comment by jantigenie** [comment]
@yurijmikhalevich You're right, AppImage version of rclip can access to mounted drives. Thanks for your work&help.

**Comment by yurijmikhalevich** [comment]
@jantigenie, glad it was resolved. I'll update you in this Issue once the permissions issue with snap is fixed. Thank you for reporting it!
